universal-login-commons/lib/core/utils/authorisation.ts

```diff
@@ -0,0 +1,24 @@
+import {utils} from 'ethers';
+
+import {CancelAuthorisationRequest} from '../models/authorisation';
+
+export const hashCancelAuthorisationRequest =
+  (cancelAuthorisationRequest: CancelAuthorisationRequest): string => {
+    const payloadString = JSON.stringify(cancelAuthorisationRequest);
+    const payloadBytes = utils.toUtf8Bytes(payloadString);
+    return utils.keccak256(payloadBytes);
+  };
+
+export const signCancelAuthorisationRequest =
+  (cancelAuthorisationRequest: CancelAuthorisationRequest, privateKey: string): utils.Signature => {
+    const payloadDigest = hashCancelAuthorisationRequest(cancelAuthorisationRequest);
+    const signingKey = new utils.SigningKey(privateKey);
+    return signingKey.signDigest(payloadDigest);
```

Comment:
We should probably wrap the last two lines into one util `sign(payload, privateKey)` function.
Also, we should find all occurrences of Wallet.sign and replace it with a newly created `sign` function if make sense.